pkg/cert/v1beta1/consts/const.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2021 The Kubeflow Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package consts
+
+const (
+	CAName  = "katib-ca"
+	Service = "katib-controller"
+	JobName = "katib-cert-generator"
+	Secret  = "katib-webhook-cert"
+	Webhook = "katib.kubeflow.org"
+)
+
+const MaxTryPatchCount = 5
```

Comment:
Keep it in the same const.
```suggestion
	Webhook = "katib.kubeflow.org"
	MaxTryPatchCount = 5
)

```

Reply:
Thanks!